libopenage/engine/engine.h

```diff
@@ -110,6 +116,11 @@ class Engine {
 	 */
 	util::Path root_dir;
 
+	/**
+	 * Window settings
+	 */
+	const renderer::window_settings window_settings;
+
```

Comment:
It's probably better to store them in the presenter class and pass them in the constructor.